### PR TITLE
[plugin.video.hdaustria@matrix] 1.0.5+matrix.1

### PR DESCRIPTION
**Pull request by michaelarnauts**
### Description

- **General**
  - Add-on name: HD Austria
  - Add-on ID: plugin.video.hdaustria
  - Version number: 1.0.5+matrix.1
  - Kodi/repository version: matrix

- **Code location**
  - URL: https://github.com/add-ons/plugin.video.tvvlaanderen

Dieses add-on erlaubt Zugriff auf Live TV Kanäle und video-on-demand Inhalte, bereitgestellt von deinem HD Austria Abonnement.

### What's new

v1.0.5 (2022-01-22)
- Fix incorrect time in EPG view.
- Fix hiding of adult channels in EPG view.
- Add translations.

### Checklist:
- [X] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [X] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-plugins/blob/master/CONTRIBUTING.md) document
- [X] Each add-on submission should be a single commit with using the following style: [plugin.video.foo] v1.0.0

### CODE REVIEW

**Comment by michaelarnauts** [comment]
This Add-on uses the same code as the TV Vlaanderen addon, but it is branded for HD Austria since the API is the same. We've decided to split it up into different add-ons, and build the ZIP with different branding and configuration so it's easier to find in the Kodi repository, this also allows to install the separately. This also makes it easier to manage in case the API's would diverge in the future.

File: plugin.video.hdaustria/resources/settings.xml [hunk 1]
```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="utf-8" standalone="yes"?>
```
Comment (enen92) [inline plugin.video.hdaustria/resources/settings.xml:1]: would be nice to get this updated to the new style settings once you have a chance to do it

**Review by enen92 — APPROVED** [review]
Looks good, code's pretty similar to other addons from the same authors

File: plugin.video.hdaustria/resources/settings.xml (hunk 1 above)
Comment (michaelarnauts) [reply plugin.video.hdaustria/resources/settings.xml:1]: Good point, I'll take a look at this later.

**Pull request merged**